cypress/integration/app_admin/app_admin.js

```diff
@@ -0,0 +1,49 @@
+import { Given, When, Then, Before, After } from 'cypress-cucumber-preprocessor/steps';
+
+const USER_NAME = 'test@unit.no';
+const APP_ADMIN_ROLE = 'App-admin';
+
+Before(() => {
+  cy.deleteUser(USER_NAME).then(() => {
+    cy.addUser(USER_NAME, APP_ADMIN_ROLE).then((userId) => {
+      cy.wrap(userId).as('userId');
+      cy.visit('/');
+      cy.wait(5000);
+
+      cy.get('[data-testid=create-author-button]').click();
+      cy.get('[data-testid=modal_next]').click();
+      cy.get('[data-testid=skip-connect-to-orcid]').click();
+      cy.get('[data-testid=menu]').click();
+
+      // Set language
+      cy.get('[data-testid=menu-user-profile-button]').click();
+      cy.get('[data-testid=language-selector]').click();
+      cy.get('[data-testid=user-language-eng]').click();
+      cy.visit('/');
+    });
+  });
+});
+
+Given('that the user is logged in', () => {
+  cy.get('@userId');
+});
+Given('they have the role of Application administrator', () => {});
+
+When('they look at any page in NVA', () => {
+  cy.visit('/');
+});
+
+Then('they see a menu containing', (tableData) => {
+  // | My Profile   |
+  // | Institutions |
+  // | Log Out      |

```

Comment:
Skal disse kommentarene være her?

Reply:
Har lagt inn tabellen fra featuren for å vise hva som ligger i den. Syns du det er en dårlig løsning?

Reply:
Skjønte ikke helt hvorfor de var der nei, men kanskje greit å gjøre det sånn hvis vi ikke har noen annen måte å linke features til tester?

Reply:
Har ikke funnet noen bedre måte.